content/api-streamer/producer.textile

```diff
@@ -0,0 +1,239 @@
+
+```[javascript]
+  var realtime = new Ably.Realtime('{{API_KEY}}');
+  var channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}');
+  setInterval(function(){ 
+      channel.publish('example', 'message data'); 
+  }, 3000);
+```
+
+```[nodejs]
+  var Ably = require('ably');
+  var realtime = new Ably.Realtime('{{API_KEY}}');
+  var channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}');
+  setInterval(function(){ 
+      channel.publish('example', 'message data'); 
+  }, 3000);
+```
+
+```[ruby]
+  realtime = Ably::Realtime.new('{{API_KEY}}')
+  channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}')
+  while true
+    channel.publish 'example', 'message data'
+    sleep 3
+  end
+```
+
+```[java]
+  AblyRealtime realtime = new AblyRealtime("{{API_KEY}}");
+  Channel channel = realtime.channels.get("{{RANDOM_CHANNEL_NAME}}");
+  new Timer().scheduleAtFixedRate(new TimerTask(){
+    @Override
+    public void run(){
+       channel.publish("example", "message data");
+    }
+  },0,3000);
+```
+
+```[csharp]
+  AblyRealtime realtime = new AblyRealtime("{{API_KEY}}");
+  IRealtimeChannel channel = realtime.Channels.Get("{{RANDOM_CHANNEL_NAME}}");
+  SetInterval(() => channel.PublishAsync("example", "message data"), TimeSpan.FromSeconds(3));
+```
+
+```[objc]
+ARTRealtime *realtime = [[ARTRealtime alloc] initWithKey:@"{{API_KEY}}"];
+ARTRealtimeChannel *channel = [realtime.channels get:@"{{RANDOM_CHANNEL_NAME}}"];
+while True:
+    [channel publish:@"example" data:@"message data"];
+    time.sleep(3)
+```
+
+```[swift]
+let realtime = ARTRealtime(key: "{{API_KEY}}")
+let channel = realtime.channels.get("{{RANDOM_CHANNEL_NAME}}")
+var publishTimer = Timer.scheduledTimer(timeInterval: 3.0, target: self, selector: #selector(ViewController.publishData), userInfo: nil, repeats: true)
+@objc func publishData(){
+ channel.publish("example", data: "message data")
+}
+```
+
+h3(#manage). Manage
+
+After you've successfully set up your app and know how to deploy your data streams into them, you can set up products to manage those data streams. Products allow you to distribute different subsets of data streams within your apps to different consumers.
+
+In your API Streamer producer dashboard's products tab, you can create new Products. While doing so, you can select the app into which you have deployed the data streams you intend to share using that Product. You can then specify a channel namespace, which is essentially a name given to the logical subset of the data in your app that you wish you share. For example, if you are publishing transport data to one of your apps called 'Public Transport Data', you could be publishing the data related to buses into the namespace called 'bus' and the data related to trains into another namespace called 'train'. You can then create two separate products, one for each of these namespaces to be distributed to the same or different sets of consumers. 
```

Comment:
Definitely need some links here to each concept, how to create things, selecting apps, etc.